typed-racket-lib/typed-racket/rep/free-variance.rkt

```diff
@@ -102,6 +102,12 @@
      (remove-frees (flip-variances inner) name)]))
 
 
+(define (make-covariant frees)
+  (combined-frees
```

Comment:
We just want delete the application of `make-covariant` so it looks like this:

```racket
[#:frees (f) (combine-frees (map f elems))]
```

similar to how the frees for [Sequence](https://github.com/racket/typed-racket/blob/master/typed-racket-lib/typed-racket/rep/type-rep.rkt#L1144) are calculated.